packages/rc-mfa-config/src/constants/urls.ts

```diff
@@ -0,0 +1,11 @@
+export const URLS = {
+  production: {
+    reapitConnectMyAccount: 'https://connect.reapit.cloud/my-account',
```

Comment:
could generate these based on the token issuer rather than use static values (thinking about anz etc. here)

Reply:
@joshbalfour I've updated this this morning. Would you mind giving this a once over again so we can get it in? Cheers